README.md

```diff
@@ -1,2 +1,119 @@
-# neo4j-http
-A simple ruby HTTP client for Neo4j
+# Neo4j::Http
+
+The `Neo4j::Http` gem provides `Neo4j::Http::Client` as a thin wrapper around the [Neo4j HTTP API](https://neo4j.com/docs/http-api/current/).
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'neo4j-http'
+```
+
+And then execute:
+
+    $ bundle install
+
+Or install it yourself as:
+
+    $ gem install neo4j-http
+
+## Configuration
+
+The client is configured by default via a set of environment variables from `[Neo4j::Http::Configuration](https://github.com/doximity/neo4j-http/blob/master/lib/neo4j/http/configuration.rb)`:
```

Comment:
You won't be able to render the link inside the backticks. It will just show a literal `[xxx](xxx)`